unittests/list_restriction/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+project(list_restriction)
+
+set(list_restriction_SRCS test_list_restriction.cpp)
+set(EXTRA_LIBS kdwsdl2cpp_lib)
```

Comment:
Unittests for the kdwsdl2cpp code directly, that's interesting. I like the idea, even though in this particular case, the standard way would have been to save those wsdl documents as actual files, running kdwsdl2cpp on them and simply checking that the result compiles, as we do in many other unittesets :-)

But I like it, it can be useful for other bugs that require direct unittests, i.e. those where "it compiles" wouldn't be enough anyway.